### ISSUE TITLE
deps/python3-*spec files should be deps/python-python3-*.spec

### ISSUE THREAD

**Issue by nkadel** [MrMEEE/awx-rpm-versions#3]
python3-saml.spec should be named python-python3-saml.spec, and its "Name:" line switched to "python-python3-saml"
python3-openid.spec should be named python-python3-openid.spec and its "Name:" line switched to "python-python3-openid"

python3-*.conf files should be renamed to match. This will help avoid confusion about what modules the SRPMs actually provide and the matching RPMs generated.

**Comment by MrMEEE** [comment]
I don't think that it's that simple.. Red Hat actually renames those packages themselves.. 

But this very well describes all of the inconsistences in the pypi repo...

**Comment by nkadel** [comment]
Your spec files produces python3.11-python3-saml and python3.11-python3-openid packages. Red Hat names them inconsistently, there's no compelling reason to emulate Red Hat's inconsistency if you're going to publish RPMs with correctly distinguished names.

**Comment by MrMEEE** [comment]
As I remember, this is actually the auto-generated names from pyp2spec.. I don't recall changing any of the manually..

**Comment by nkadel** [comment]
I believe you. It's an underlying pyp2spec bug.
